ci/publish-oci.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+if [ ! -f "${HOME}/.unifi/bin/unifi" ]; then
+  echo "Unifi not found! Exit"
+  exit 1
+fi
+
+if [[ "$#" != 3 ]]; then
+ echo "No enjoy parameters!"
+ echo "Usage: $(basename $0) <staging_dir> <release_tag> <registry>"
+ exit 1
+fi
+
+STAGING_DIR=$1
+RELEASE_TAG=$2
+REGISTRY=$3
+
+info() {
+  (>&2 printf "\e[90m$(basename $0)\e[97m: [\e[0;36mINFO\e[97m]:\e[0m $1")
+}
+info_done() {
+  (>&2 printf "\t\t\e[97m[\e[1;32mDONE\e[97m]\e[0m\n")
+}
+info_fail() {
+  (>&2 printf "\t\t\e[97m[\e[1;31mFAIL\e[97m]\e[0m\n")
+}
+on_exit() {
+  info "Cleanup...\t\t\t"
+  rm -rf $STAGING_DIR/dist && info_done
+}
+
+trap on_exit EXIT
+
+gen_manifest() {
+local dest=$1
+info "Generate manifest...\t\t"
+echo 'spec:
+  commands:
+  - path: "damlc"
+    name: damlc
+    desc: "damlc - Compiler and IDE backend for the Daml programming language"
+' > $dest && info_done
+}
+
+pushd $STAGING_DIR 2>&1 >/dev/null
```

Comment:
If you want to hide all output, you need to put 2>&1 after >/dev/null. Feel free to ignore this comment if you intended to replace stdout with stderr.